int16 state
int16 mode

int16 targetx
int16 targety
int16 targett

int16 ballposx
int16 ballposy

int16 balldetect
int16 localizationflag

int16 localizationposx
int16 localizationposy
int16 localizationpost